docs/pages.jl

```diff
@@ -22,6 +22,7 @@ pages = Any[
         "Examples" => Any[
             "model_creation/examples/basic_CRN_library.md",
             "model_creation/examples/programmatic_generative_linear_pathway.md",
+            "model_creation/examples/noise_modelling_approaches.md",
```

Comment:
I would put this second to last in the list. That would keep the ODE-based models next to each other, and to some degree I think they are simpler examples than this one.

Reply:
sounds good, will update